airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.tsx

```diff
@@ -74,7 +74,12 @@ export const ConnectorDocumentationLayout: React.FC = ({ children }) => {
         </ReflexSplitter>
       )}
       {screenWidth > 500 && documentationPanelOpen && (
-        <ReflexElement className="right-pane" size={1000} propagateDimensions minSize={60}>
+        <ReflexElement
+          className={classNames("right-pane", styles.rightPanelStyle)}
```

Comment:
Is this "right-pane" class name necessary for reflex?

Reply:
Good question - I just tried removing them and I didn't notice any changes. They are not referenced anywhere else in our code, and I'm not seeing anything in the Reflex documentation that says these class names are required.

I noticed that those `right-pane` and `left-pane` class names are used in the reflex demo example here: https://leefsmp.github.io/Re-Flex/index.html
So my guess is that these just came from a copy-paste from that example, and are therefore unnecessary.

Reply:
Removed those class names